app/templates/courses/teacher-class-view.jade

```diff
@@ -1,193 +1,3 @@
-          h4.m-b-2
-            | &nbsp;
-          div.small-details.rtl-allowed
-            if state.get('earliestIncompleteLevel')
-              span(data-i18n='teacher.earliest_incomplete')
-              span :
-              +longLevelName(state.get('earliestIncompleteLevel'))
-              +inlineUserList(state.get('earliestIncompleteLevel').users)
-            else if (view.classroom.get('members') || []).length > 0 && !view.allStatsLoaded()
-              span(data-i18n='teacher.earliest_incomplete')
-              span :
-              div(data-i18n='common.loading')
-
-          div.small-details.m-t-3.rtl-allowed
-            if state.get('latestCompleteLevel')
-              span(data-i18n='teacher.latest_complete')
-              span :
-              +longLevelName(state.get('latestCompleteLevel'))
-              +inlineUserList(state.get('latestCompleteLevel').users)
-            else if (view.classroom.get('members') || []).length > 0 && !view.allStatsLoaded()
-              span(data-i18n='teacher.latest_complete')
-              span :
-              div(data-i18n='common.loading')
-
-        .adding-students.col-md-5
-          h4.m-b-2
-            span(data-i18n='courses.add_students')
-            span :
-          +copyCodes
-          +addStudentsButton
-
-      if view.students.length > 0
-        ul.nav.nav-tabs.m-t-5(role='tablist')
-          - var activeTab = state.get('activeTab');
-          li(class=(activeTab === "#students-tab" ? 'active' : ''))
-            a.students-tab-btn(href='#students-tab')
-              .small-details.text-center(data-i18n='courses.students')
-          .tab-spacer
-          li(class=(activeTab === "#course-progress-tab" ? 'active' : ''))
-            a.course-progress-tab-btn(href='#course-progress-tab')
-              .small-details.text-center(data-i18n='teacher.course_progress')
-          .tab-spacer
-          if view.classroom.hasAssessments()
-            li(class=(activeTab === "#assessments-tab" ? 'active' : ''))
-              a.assessments-tab-btn(href='#assessments-tab')
-                .small-details.text-center(data-i18n='teacher.assessments')
-            .tab-spacer
-          li(class=(activeTab === "#license-status-tab" ? 'active' : ''))
-            a.course-progress-tab-btn(href='#license-status-tab')
-              .small-details.text-center(data-i18n='teacher.license_status')
-          .tab-filler
-
-        .tab-content
-          if activeTab === '#students-tab'
-            +studentsTab
-          else if activeTab === '#course-progress-tab'
-            +courseProgressTab
-          else if activeTab === '#assessments-tab'
-            +assessmentsTab
-          else if activeTab === '#license-status-tab'
-            +enrollmentStatusTab
-
-      else
-        .text-center.m-t-5.m-b-5
-          .text-h2
-            span(data-i18n="teacher.no_students_yet")
-          .text-h4
-            span(data-i18n="teacher.try_refreshing")
-
-mixin breadcrumbs
```

Comment:
Why did we remove this mixin?

Reply:
Woah - good catch - this drops breadcrumbs from normal teacher class view and its not added back.  Will add

Reply:
Added back